applications/OkAlice-Liquid-Staking.md

```diff
@@ -0,0 +1,153 @@
+
+
+
+## Development Status :open_book:
+
+## Development Roadmap :nut_and_bolt:
+### Overview 
+
+- **Total Estimated Duration:** 5 months
+- **Full-Time Equivalent (FTE):**  1.5
+- **Total Costs:** 55,000 USD
+
+### Milestone 1: Validator Selector
+
+- **Estimated Duration:** 2 months 
+- **FTE:**  1.5 
+- **Costs:** 22,000 USD 
+
+| Number | Deliverable | Specification |
+| ------:| ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | Inline documentation of the code and a basic tutorial on using the Substrate nodes and test transactions |
+| **0c.** | Testing and Testing Guide | Comprehensive unit tests and a guide on running these tests |
+| **0d.** | Docker | Dockerfile(s) for testing the milestone functionality |
+| 1. | XCM CE | Implementation of XCM chain-extension for nomination pool manipulation |
+| 2. | Oracle | Implementation of a simple oracle for publishing validator statistics |
+| 3. | Oracle prov | Backend scripts for provisioning oracle data |
+| 4. | Validator Selector | Contract implementation for optimized selection of nomination pool based on oracle's published statistics |
+
+### Milestone 2: Issuer-Staker
+
+- **Estimated Duration:** 2 months 
+- **FTE:**  1.5 
+- **Costs:** 22,000 USD 
+
+| Number | Deliverable | Specification |
+| ------:| ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | Inline documentation of the code and a basic tutorial on using the Substrate nodes and test transactions |
+| **0c.** | Testing and Testing Guide | Comprehensive unit tests and a guide on running these tests |
+| **0d.** | Docker | Dockerfile(s) for testing the milestone functionality |
+| 1. | Issuer-Staker | Contract implementation providing PSP22 interface for the derivative token and managing staking operations |
+| 2. | Staking UI | End-user UI implementation for staking operations | 
+
+### Milestone 3: Bootstrap phase
+
+- **Estimated Duration:** 2 weeks
+- **FTE:**  1.5 
+- **Costs:** 11,000 USD 
+
+| Number | Deliverable | Specification |
+| ------:| ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | Inline documentation of the code and a basic tutorial on using the Substrate nodes and test transactions |
+| **0c.** | Testing and Testing Guide | Comprehensive unit tests and a guide on running these tests |
+| **0d.** | Docker | Dockerfile(s) for testing the milestone functionality |
+| **0e.** | Article | Article explaining the implementation of liquid staking in an Ink! environment |
+| 1. | XCM CE | Replacement of the CE with Ink! native XCM functions
+| 2. | Deploy | Deployment of the contract on Shibuya (Westend) |
+| 3. | Documentation | Online documentation detailing the functioning of the first pool instance |
+| 4. | Documentation | Online documentation for deploying your own strategy for managing the pool |
+| 5. | Audit | External organization audit of the contract |
```

Comment:
ink!ubator has two Audit companies in the Auditor Track. One of them will provide these services for you

Reply:
Thanks, that's great news!